src/plugins/en.json

```diff
@@ -78,23 +78,23 @@
     "dataQualityImplementation8": "A process has been developed to manage any risks of having outdated or unreliable data that could be used in the system.",
     "dataQualityImplementation9": "This information is now publicly available.",
     "dataQualityImplementation10": "The data used for the system has been posted on the Open Government portal.",
-    "proceduralFairnessImplementation1": "The audit trail clearly identifies the authority or delegated authority as identified in legislation.",
-    "proceduralFairnessImplementation2": "The system records all the recommendations or decisions made by the system.",
-    "proceduralFairnessImplementation3": "All key decision points are identifiable in the audit trail.",
-    "proceduralFairnessImplementation4": "All key decisions points are linked to the relevant legislation, policy or procedure.",
-    "proceduralFairnessImplementation5": "A change log has been developed to detail all of the changes made to the model and to the system.",
-    "proceduralFairnessImplementation6": "The audit trail identifies which version of the system was used for each decision.",
-    "proceduralFairnessImplementation7": "The system's audit trail indicates all of the decision points made by the system.",
-    "proceduralFairnessImplementation8": "The audit trail generated by the system can be used to generate notifications, including a statement of reason.",
-    "proceduralFairnessImplementation9": "The system's audit trail shows who the authorized decision-makers are.",
-    "proceduralFairnessImplementation10": "The system is able to produce reasons for its decision or recommendations when required.",
-    "proceduralFairnessImplementation11": "There is a process in place to grant, monitor, and revoke access permission to the system.",
-    "proceduralFairnessImplementation12": "There is a mechanism to capture feedback by users of the systems.",
-    "proceduralFairnessImplementation13": "There is a recourse process in place for clients that wish to challenge the decision.",
-    "proceduralFairnessImplementation14": "The system enables human override of system decisions.",
-    "proceduralFairnessImplementation15": "There is a process in place to log instances when overrides were performed.",
-    "proceduralFairnessImplementation16": "The system's audit trail includes change control processes to record modifications to the system's operation or performance.",
-    "proceduralFairnessImplementation17": "A concept case has been prepared and approved by the Government of Canada Enterprise Architecture Review Board.",
+    "fairnessImplementation1": "The audit trail clearly identifies the authority or delegated authority as identified in legislation.",
```

Comment:
Fix the key names, and align with survey-en-fr.json, where the order was different.